client/me/purchases/manage-purchase/payment-method-selector/style.scss

```diff
@@ -48,7 +48,9 @@
 }
 
 .payment-method-selector__all-subscriptions-checkbox-label {
-	margin-bottom: 1.5em;
+	&.form-label {
```

Comment:
As above, we can probably revert this for now as the checkbox will be removed in https://github.com/Automattic/wp-calypso/pull/56736